hermes-common/src/main/java/pl/allegro/tech/hermes/common/kafka/offset/PartitionOffsets.java

```diff
@@ -0,0 +1,26 @@
+package pl.allegro.tech.hermes.common.kafka.offset;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class PartitionOffsets implements Iterable<PartitionOffset> {
```

Comment:
PartitionOffsets was moved from pl.allegro.tech.hermes.domain.subscription.offset.PartitionOffsets and changed quite a bit, so git reports in as file added and removed (below).

I moved this to common.kafka, because that's where all other kafka-related classes were located and I wanted them to be close together. We should move this whole common.kafka package to infra.kafka IMO, but maybe not as part of this huge PR :)